conf/configure.fv3_coupled.gaea.intel

```diff
@@ -66,6 +64,10 @@ else
 CPPDEFS += -DMOIST_CAPPA -DUSE_COND
 endif
 
+ifeq ($(NAM_phys),Y)
```

Comment:
NAM_phys was removed from the fv3, so please remove this if statement.

Reply:
If we remove these, this means that we will be out of sync with the ufs-weather-model/conf/ files.  I just checked and the NAM_phys option is still in https://github.com/ufs-community/ufs-weather-model/blob/develop/conf/configure.fv3.gaea.intel and https://github.com/ufs-community/ufs-weather-model/blob/develop/conf/configure.fv3.gaea.intel    In the past, we kept the s2s model in sync with the weather model except if you do a diff between the fv3 and fv3_coupled you would get: 
179c179
< CPPFLAGS += -DFV3_CPLD -traditional
---
> CPPFLAGS += -traditional

If you would like these removed, I can remove them from all of the conf files though.

Reply:
ufs-weather-model removes this in conf files in tier1 supported platforms. It will be removed when the code is tested in  tier 2 platforms. It's up to you to remove it in all the conf, anyway not sure if you want to add this option back in gaea config why ufa-weather is going to remove